### ISSUE TITLE
Two color changes to white 

### ISSUE THREAD

**Issue by n3roGit** [mwittig/pimatic-milight-reloaded#13]
Hiho, 

If I try to change the color of two milight zones in one rule only the first is triggered. 

Set color milight-rgbw-zone-1 to white and dim milight-rgbw-zone-1 to 80 and milight set color milight-rgbw-zone-2 to white and dim milight-rgbw-zone-2 to 80

**Comment by n3roGit** [comment]
Found a solution. You need to add a delay between the different zones to change color. Is it possible to add this to the plugin?

**Comment by mwittig** [comment]
Which Milight Wifi Controller do you have? Is it the new one, i.e. iBox1 / iBox2 or is it the old one?

**Comment by n3roGit** [comment]
I have the new one.

**Comment by mwittig** [comment]
I have released **pimatic-milight-reloaded@0.9.12** including a new version of the base driver which now supports synchronization accross multiple instances.  Moreover, it contains several bug fixes for the setColor method used as part of the milight set color action.
Can you please let mee know if this works any better for you? I have tested with following which works will for me.

```
milight set effectMode Milight V6 Full Color Zone mode 1 and milight set effectMode Milight V6 Bridge Light mode 1 and milight set color Milight V6 Full Color Zone to yellow after 3s and milight set color Milight V6 Bridge Light to yellow after 3s

**Comment by n3roGit** [comment]
Yeah. I have to test it a little bit more but looks great now =) THX

**Comment by mwittig** [comment]
Please keep me posted. Meanwhile, I have published pimatic-milight-reloaded@0.9.14 with further bug fixture. Moreover, the are some notable changes with regards to the default behavior.

 ## Notable Changes

As of version 0.9.14 the color device types now implement the default behaviour of pimatic dimmer, i.e. turning the dimmer on will always set the dim level to 100%. If you wish to retain the old behavior, you set the device configuration option `keepDimlevel`  to `true`.

**Comment by mwittig** [comment]
I assume the issue has been resolved, now. Please re-open otherwise
